packages/parser/src/build/index.ts

```diff
@@ -1,39 +1,26 @@
-import { isTokenMatch } from '@terrazzo/token-tools';
+import type { DocumentNode } from '@humanwhocodes/momoa';
```

Comment:
Believe it or not, moved to `.ts`. It started being _heavily_ untyped. But with all the rule additions, the scales tipped in favor of stronger, more-predictable types. It even caught a few untested bugs 🙃